app/templates/app/search.html

```diff
@@ -6,81 +6,161 @@
+                                               for="{{ checkbox.id_for_label }}">
+                                            {{ checkbox.choice_label }}
+                                        </label>
+                                    </div>
+                                {% endfor %}
                             </div>
                         </div>
-                    </form>
-                    <div class="mb-3">
-                        <span class="mr-5"><b>{% trans "Results:" %}</b></span>
-                        <span>{{ documents|length }}</span>
                     </div>
-                    <div>
-                        {% for document in documents %}
-                            <div>
-                                <ul>
-                                    <li>
-                                        <span class="mr-5 text-primary"><b>{% trans "Title:" %}</b></span>
-                                        <span>{{ document.title }}</span>
-                                    </li>
-                                    <li class="mb-2">
-                                        <span class="mr-5"><b>{% trans "Institution:" %}</b></span>
-                                        <span>{{ document.institution }}</span>
-                                    </li>
-                                    <li class="mb-2">
-                                        <span class="mr-5"><b>{% trans "Headline:" %}</b></span>
-                                        <span>{{ document.search_headline|safe }}</span>
-                                    </li>
-                                    <li>
-                                        <span class="mr-5"><b>{% trans "File:" %}</b></span>
-                                        <span>{{ document.uploaded_file }}</span>
-                                    </li>
-                                    <li>
-                                        <span class="mr-5"><b>{% trans "License:" %}</b></span>
-                                        <span>{{ document.license }}</span>
-                                    </li>
-                                    <li>
-                                        <span class="mr-5"><b>{% trans "Mime Type:" %}</b></span>
-                                        <span>{{ document.mime_type }}</span>
-                                    </li>
-                                    <li class="mt-2">
-                                        <span class="mr-5"><b>{% trans "Rank:" %}</b></span>
-                                        <span><i>{{ document.rank }}</i></span>
-                                    </li>
-                                </ul>
+                    <div class="col-md-3">
+                        <div class="form-group">
+                            <label for="id_subjects">Document Type</label>
+                            <div class="subjects-checkbox-container">
```

Comment:
Please consider a more general name for the container, as this is not about subjects.